tests/test_bestscore.py

```diff
@@ -0,0 +1,68 @@
+from autopredictor.bestscore import display_best_score
+import pandas as pd
+import pytest
+
+def test_existing_scoring_metric():
+    """Test that the function returns the correct DataFrame for an existing scoring metric."""
+    df = pd.DataFrame({'MAE': [5.6, 3.4],
+                           'MSE': [9.4, 21.4],
+                           'MAPE': [0.34, 0.45]},
+                           index=['Linear Regression', 'Random Forest'])
+    
+    expected_output = pd.DataFrame({
+        'MAE': [5.6]
+    }, index=['Linear Regression'])
+
+    result = display_best_score(df, 'MAE')
+
+    pd.testing.assert_frame_equal(result, expected_output)
+
+def test_invalid_scoring_metric():
+    """Test that the function raises the correct error message for a invalid scoring metric."""
+    df = pd.DataFrame({'MAE': [5.6, 3.4],
+                       'MSE': [9.4, 21.4],
+                       'MAPE': [0.34, 0.45]},
+                       index=['Linear Regression', 'Random Forest'])
+
+    invalid_metric = 'R2'
+    available_metrics = df.columns.tolist()
+    expected_error = f"Invalid Scoring metric '{invalid_metric}'.The specified metric is not in the list of available metrics. Available metrics: {', '.join(available_metrics)}."
```

Comment:
```suggestion
    expected_error = f"Invalid Scoring metric '{invalid_metric}'. The specified metric is not in the list of available metrics. Available metrics: {', '.join(available_metrics)}."
```
Grammar (if changed here, it must be changed in the function file)